packages/core/lib/testing/Test.js

```diff
@@ -10,10 +10,10 @@ const WorkflowCompile = require("@truffle/workflow-compile");
 const Resolver = require("@truffle/resolver");
 const TestRunner = require("./TestRunner");
 const SolidityTest = require("./SolidityTest");
-const RangeUtils = require("@truffle/compile-solidity/compilerSupplier/rangeUtils");
+const RangeUtils = require("@truffle/compile-solidity/dist/compilerSupplier/rangeUtils");
```

Comment:
Similarly with these two.

Reply:
(Resolving this to move discussion to [other comment thread](https://github.com/trufflesuite/truffle/pull/4223#discussion_r688190965))